app/src/main/res/layout/fragment_bookmarks.xml

```diff
@@ -6,32 +6,34 @@
     android:layout_height="match_parent"
     tools:context=".MainActivity">
 
-    <com.airbnb.epoxy.EpoxyRecyclerView
-        android:id="@+id/epoxyRecyclerView"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
+    <androidx.swiperefreshlayout.widget.SwipeRefreshLayout
+        android:id="@+id/swipeRefreshLayoutBookmarks"
```

Comment:
One last change here is that we prefer to append the view type to its name for consistency so the views ids are always like (`name`) + `type` or in this case `android:id="@+id/bookmarksSwipeRefreshLayout"` or simply `android:id="@+id/swipeRefreshLayout"`

Reply:
Oh, ok.